src/main/java/it/gov/pagopa/pu/sil/security/SecurityUtils.java

```diff
@@ -67,4 +68,15 @@ public static String removePiiFromURI(URI uri){
       : null;
   }
 
+  public static boolean isValidWsSilUser(String organizationIpaCode) {
```

Comment:
This method is checking if the user has ADMIN role on the input organization
```suggestion
  public static boolean isAdminUser(String organizationIpaCode) {
```

Reply:
done